frontend/src/components/header/header.tsx

```diff
@@ -82,7 +83,12 @@ export const Header = (): React.ReactElement => {
     },
     logout: {
       label: 'Logout',
-      onClick: () => navigate('#'),
+      onClick: (): void => {
+        logOut();
+        //A temporary solution until logout is implemented
+        localStorage.removeItem('token');
+        location.reload();
```

Comment:
This functionality was already implemented as a middleware for reducer `logOut` in PR 151